<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Forma</class>
 <widget class="QWidget" name="Forma">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>478</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Изменение записи</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="Error_lable">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>433</y>
     <width>301</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Update_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>105</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
    border-style: outset;
    border-width: 2px;
    border-color: gray;
    padding: 4px;

        background-color: #25F7F2;
	border-radius: 10px;


}

QPushButton:pressed {
    border-style: inset;
     background-color: #25F7A2;
}
</string>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>23</y>
     <width>291</width>
     <height>224</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="Name_label">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Имя</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="Name">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #D3D3D3;
    border: 2px solid #00CED1;
    border-radius: 5px;
}
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Target_label">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Курс</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QLineEdit" name="Target">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #D3D3D3;
    border: 2px solid #00CED1;
    border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>Просто</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="Contact_label">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Контакты</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLineEdit" name="Contact">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #D3D3D3;
    border: 2px solid #00CED1;
    border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>+79601253072</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="Cost_label">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Стоимость</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QLineEdit" name="Cost">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #D3D3D3;
    border: 2px solid #00CED1;
    border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QLabel" name="Date_label">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Дата начала</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="Date">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #D3D3D3;
    border: 2px solid #00CED1;
    border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>2023-07-04</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="ID_label">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>ID записи</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2">
     <widget class="QLineEdit" name="ID">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #D3D3D3;
    border: 2px solid #00CED1;
    border-radius: 5px;
}
</string>
      </property>
      <property name="text">
       <string>213</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>284</y>
     <width>286</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="Date_label_5">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Первый день</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QComboBox" name="WeekDaybox1">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::NoInsert</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContents</enum>
      </property>
      <property name="duplicatesEnabled">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>ПН</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ВТ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>СР</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ЧТ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ПТ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>СБ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ВС</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QTimeEdit" name="FirstDayTime">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTimeEdit {
    background-color: #D3D3D3;
    border: 2px solid #00CED1;
    border-radius: 5px;
}
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QCheckBox" name="check2">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2 раза в неделю?</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Date_label_2">
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Второй день</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QComboBox" name="WeekDaybox2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>ПН</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ВТ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>СР</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ЧТ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ПТ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>СБ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ВС</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QTimeEdit" name="SecondDayTime">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTimeEdit {
    background-color: #D3D3D3;
    border: 2px solid #00CED1;
    border-radius: 5px;
}
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
